extensions/Worker.Extensions.Http.AspNetCore.Analyzers/src/CodeFixForHttpResultAttributeExpected.cs

```diff
@@ -64,8 +64,13 @@ protected override async Task<Document> GetChangedDocumentAsync(CancellationToke
 
                 var typeNode = root.FindNode(this._diagnostic.Location.SourceSpan)
                     .FirstAncestorOrSelf<TypeSyntax>();
-
                 var typeSymbol = semanticModel.GetSymbolInfo(typeNode).Symbol;
+
+                if (SymbolUtils.TryUnwrapTaskOfT(typeSymbol, semanticModel, out var innerSymbol))
+                {
+                    typeSymbol = innerSymbol;
+                }
+
                 var typeDeclarationSyntaxReference = typeSymbol.DeclaringSyntaxReferences.FirstOrDefault();
```

Comment:
typeSymbol can be null here, right? as it is returned by method TryUnwrap(). In that case, we will need a null check here

Reply:
innerSymbol is not null if `TryUnwrapTaskOfT` returns true so it would not be `null` at this line.